src/animation/AnimationAction.js

```diff
@@ -76,7 +76,7 @@ function AnimationAction( mixer, clip, localRoot ) {
 
 }
 
-AnimationAction.prototype = {
+Object.assign( AnimationAction.prototype, {
 
 	constructor: AnimationAction,
```

Comment:
Yes we could remove the constructor line, it is not necessary now. I will give it a try to be sure there is no hidden regression, and propose a new PR.